components/backup-stream/src/service.rs

```diff
@@ -13,11 +13,11 @@ use crate::{
 };
 
 #[derive(Clone)]
-pub struct Service {
+pub struct BackupStreamGrpcService {
```

Comment:
Perhaps don't rename it? As other components (`cdc`, `importing`) uses the name `Service` as gRPC service too.

Reply:
oh I see. yeah I would change their names later. the `Service` is a bit ambiguous, especially we have other components that use the same name. In our case, it could be gprc service or the service of the entire backup stream, if we name `endpoint` to be `service` it actually makes sense as well. People will need to dig into the code to figure out what the `service` really means and it sort of takes time.
I would argue the naming should be self explanatory enough that people don't need to read the code to figure out what it does. If it's doing too many things and it would be too long for the name, I would explore if we can 1. divide it into smaller components and 2. add enough comments. 
we can discuss more on the standup!